Apps/ValidationTests/Shared/TestUtils.h

```diff
@@ -29,6 +29,12 @@ namespace
     int errorCode{};
 }
 
+// can't externalize variable with ObjC++. Using a function instead.
+int GetExitCode()
+{
+    return errorCode;
+}
```

Comment:
where is this used?

Reply:
in MacOS appdelegate. I forgot to push that change.